yapftests/format_token_test.py

```diff
@@ -32,6 +33,16 @@ def testSimple(self):
     self.assertEqual("FormatToken(name=COMMENT, value=# A comment)", str(tok))
     self.assertTrue(tok.is_comment)
 
+  def testIsMultilineString(self):
```

Comment:
This should go into `reformatter_python3_test.py`. It has support for PY3 stuff.